src/jit/morph.cpp

```diff
@@ -7243,7 +7244,7 @@ bool Compiler::fgCanFastTailCall(GenTreeCall* callee)
     }
 
     // Callee has a >8 and <=16 byte struct and arguments that has to go on the stack. Do not fastTailCall.
-    if (calleeStackSize > 0 && hasTwoSlotSizedStruct)
+    if (hasStackArgs && hasTwoSlotSizedStruct)
```

Comment:
Should this comment say "caller or callee"? In the bug, it's the caller that has a multi-slot stack arg.

You might add a reason why, some thing like

When either caller or callee has multi-slot stack args, we can't safely reason about stack interference between the two with arg numbers, we need to look at stack slot usage. And we don't do this yet.

Reply:
Yes this comment is now outdated. Thank you will change.